src/briefcase/integrations/wix.py

```diff
@@ -73,9 +73,11 @@ def verify(cls, command, install=True):
             not installed, raises MissingToolError.
         """
         if command.host_os != 'Windows':
-            raise BriefcaseCommandError("""
+            raise BriefcaseCommandError(
+                """
```

Comment:
I think some of the errors in this file may need the `\` trick to avoid extra newlines (plus also auditing the newlines at the end of each message)

Reply:
This file is compact so I added line continuations to the `BriefcaseCommandError` exceptions to avoid the extra initial new line.